rpcd/playbooks/roles/rpc_maas/templates/network_throughput.yaml.j2

```diff
@@ -1,4 +1,4 @@
-{% set max_speed = (item.0.max_speed | default(0, boolean=true) or item.1.stdout) | int * 10 ** 6 / 8 %}
+{% set max_speed = (item.0.max_speed | default(item.1.stdout | default(1000, boolean = true), boolean = true)) | int * 10 ** 6 / 8 %}
```

Comment:
Yes, auto-detect is the default for a production deployment. What your value of 1000 does is hide potential failure, it's essentially simplistic error handling. The only reason auto-detect should fail is if the deployment is on virtualised hardware, for those situations the variable net_max_speed exists so that a value can be specified in the overrides file.